parts/linux/cloud-init/artifacts/cgroup-stats.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Update cgroup statisitics to Kusto on a 5 minute timer
```

Comment:
nit: drop Kusto, just say something generic like "emit telemetry"

(internal codename, but they rebranded too late and it stuck. technical Azure Data Explorer but not really relevant anyway)